sentry_streams/sentry_streams/deployment_config/alerts_config.yaml

```diff
@@ -0,0 +1,37 @@
+env:
+  topics:
+    logical-events: events
+    transformed-events: transformed-events
+    transformed-events-2: transformed-events-2
+
+# These are intended to be runtime-agnostic config
+pipeline:
+  sources_config:
+    myinput:
+      bootstrap_servers: localhost:9092
+      consumer_group: python-test
+      auto_offset_reset: latest
+
+  sinks_config:
+    kafkasink:
+      bootstrap_servers: localhost:9092
+
+  reduce_config:
+    myreduce:
+      parallelism: 3
+
+flink:
```

Comment:
This is meant to be for runtime-specific overrides. I'd like to override the fields that are defined in this section -- wanted to use Pydantic for some utils that make operations like deep updates of dictionaries easier, but I can't manage to install it.